app/code/Magento/Inventory/Test/Acceptance/Test/AdminStockCannotBeDeletedOnStockPageWhenSalesChannelIsAssignedTest.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="../../../../../../../dev/tests/acceptance/vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Test/etc/testSchema.xsd">
+    <test name="AdminStockCannotBeDeletedOnStockPageWhenSalesChannelIsAssignedTest">
+        <annotations>
+            <features value="Multi-Source Inventory"/>
+            <stories value="Cannot Delete Stock from Stock Edit Page when Sales Channel assigned"/>
+            <title value="Cannot Delete Stock from Stock Edit Page when Sales Channel assigned"/>
+            <description value="You should not be able to delete a Stock fro Stock Edit Page when a Sales Channel is assigned."/>
+            <testCaseId value="950"/>
+            <severity value="CRITICAL"/>
+            <group value="msi"/>
+        </annotations>
+
+        <before>
+            <createData entity="BasicMsiStock1" stepKey="createStock1"/>
```

Comment:
Please try to avoid numbers in `stepKeys`
Make them more descriptive like `stepKey="createBasicStock"` , `stepKey="createCustomStock"`, `stepKey="createStockWithSpecialValues"`, etc